docs/viplab3.0/computation_template.md

```diff
@@ -243,23 +275,28 @@ Four access levels can be specified inside a part:
 
 ### JSON object Parameter
 
-An PARAM_ID-object, like *\_\_BINARY\_\_*, has the following members:
+A parameter-object, has the following members:
 
 |Key [--Subkey] | Type (a default is marked by _italics_)|Opt / Must |Description |Comment | 
 |---------------|----------------------------------------|-----------|------------|--------|
-|guiType | one of {*"editor"*, "input_field", "checkbox", "radio", "dropdown", "toggle", "slider"} | must | specifies how the frontend renders the parameter | |
-|name | string | must | Label for the parameter | frontend feature |
-|type | one of {"number", "text"} | opt | Type of the input field |  | 
+|type | one of {"any-type", "fixed-type"} | must | specifies type of the parameter | depends on the guiType - *fixed-type*: checkbox, radio, dropdown, toggle; *any-type*: slider, input_field, editor |
```

Comment:
The reference to the ui element should be removed, as this parameter is used to define the type of validation that is performed